docs/examples/ai/resources/runDir/run_this.sh

```diff
@@ -1,3 +1,6 @@
 aws codeartifact login --tool pip --domain superai --repository pypi-superai-internal
```

Comment:
Won't work with the aws credentials that you get with `superai login`

Reply:
Will change this